terraform/global/um-pep-engine/config/pep-engine.tf

```diff
@@ -96,18 +101,41 @@ resource "kubernetes_deployment" "pep-engine" {
       spec {
 
         automount_service_account_token = true
-
+  
         volume {
-          name = "vol-userman"
-
+          name = "pep-engine-logs"
           persistent_volume_claim {
-            claim_name = "eoepca-userman-pvc"
+            claim_name = "pep-engine-logs-volume-claim"
+          }
+        }
+        volume {
+          name = "pep-engine-lib-ext"
+          persistent_volume_claim {
+            claim_name = "pep-engine-lib-ext-volume-claim"
+          }
+        }
+        volume {
+          name = "pep-engine-custom-static"
+          persistent_volume_claim {
+            claim_name = "pep-engine-custom-static-volume-claim"
+          }
+        }
+        volume {
+          name = "pep-engine-custom-pages"
+          persistent_volume_claim {
+            claim_name = "pep-engine-custom-pages-volume-claim"
+          }
+        }
+        volume {
+          name = "mongo-persistent-storage"
+          persistent_volume_claim {
+            claim_name = "mongo-persistent-storage-volume-claim"
```

Comment:
Revert this back to the strategy used in develop (single PVC and Persistence Volume with dedicated path for mongo)